build/pkgs/sympy/package-version.txt

```diff
@@ -1 +1 @@
-1.12.1
+1.13.0
```

Comment:
I am assuming that this is what it should be set to now...

There is likely to be a 1.13.1 release coming very soon.